test/record/record-requestSpec.js

```diff
@@ -75,11 +75,12 @@ describe( 'records are requested from cache and storage sequentually', function(
 		expect( options.storage.lastRequestedKey ).toBe( 'doesNotExist' );
 	});
 
-	it( 'fails gracefully if an error occured out of order', function( done ){
+	//TODO: yasserf: I don't get this test
```

Comment:
it destroys the record request in between getting data from the cache and from the storage - which is not supposed to happen...but just in case